.github/workflows/publish_wiki.yaml

```diff
@@ -0,0 +1,52 @@
+name: Publish Wiki
+
+on:
+  push:
+    paths:
+      - .github/wiki/**
+    branches:
+      - develop
+      - chore/add-template-development-docs
+
+env:
+  USER_TOKEN: ${{ secrets.WIKI_ACTION_TOKEN }}
+  USER_NAME: github-wiki-workflow
+  USER_EMAIL: dev@nimblhq.co
+  OWNER: ${{ github.event.repository.owner.name }}
+  REPOSITORY_NAME: ${{ github.event.repository.name }}
+  WIKI_FOLDER: .github/wiki
+  TEMP_CLONE_WIKI_FOLDER: tmp_wiki
+
+jobs:
+  publish:
+    name: Publish Wiki
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
+
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.8.0
```

Comment:
We can use `uses: nimblehq/github-actions-workflows/.github/workflows/publish_wiki.yml@0.1.0` now @byhbt :D